lib/core/test/integration/Test/Integration/Scenario/CLI/Transactions.hs

```diff
@@ -204,18 +207,18 @@ spec = do
                 ]
 
         -- post transaction
-        (c, out, err) <- postTransactionViaCLI ctx "cardano-wallet" args
+        (c, out, err) <- postTransactionViaCLI @t ctx "cardano-wallet" args
         err `shouldBe` "Please enter a passphrase: **************\nOk.\n"
         txJson <- expectValidJSON (Proxy @(ApiTransaction t)) out
         verify txJson
-            [ expectCliFieldBetween amount (feeMin + amt, feeMax + amt)
+            [ expectCliFieldBetween amount (feeMin + (2*amt), feeMax + (2*amt))
```

Comment:
So, this was left unnoticed because the fee amounts are much bigger on the bridge scenarios and make the amount completely insignificant. This isn't the case with Jörmungandr where the amount and fee are more of less of the same order of magnitude!